fuel/app/tests/helper/common.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * @group Helper
+ */
+class Test_Helper_Common extends TestCase
+{
+	/**
+	 * @before
+	 */
+	public function truncate_data()
+	{
+		DBUtil::truncate_table('messages');
+	}
+
+	public function test_format_time()
+	{
+		$expect = "10/05/2015 12:10";
+		$date   = new DateTime($expect);
+		$date   = $date->getTimestamp();
+
+		$this->assertEquals(Helper_Common::format_time($date), $expect);
+	}
+
+	/**
+	 * @dataProvider data_insert_data
+	 */
+	public function test_insert_data($input,$expect)
```

Comment:
@luong-phuc-mulodo ah.. i understood. you use 'dataProvider' as annotation. that's good!